libuavcan/include/uavcan/transport/can_acceptance_filter_configurator.hpp

```diff
@@ -0,0 +1,90 @@
+/*
+ * Copyright (C) 2014 Pavel Kirienko <pavel.kirienko@gmail.com>,
+ *                    Ilia  Sheremet <illia.sheremet@gmail.com>
+ */
+
+#ifndef CAN_ACCEPTANCE_FILTER_CONFIGURATOR_HPP_INCLUDED
+#define CAN_ACCEPTANCE_FILTER_CONFIGURATOR_HPP_INCLUDED
```

Comment:
Proper guard name would be `UAVCAN_TRANSPORT_CAN_ACCEPTANCE_FILTER_CONFIGURATOR_HPP_INCLUDED` (the current one lacks prefix).